src/wallet/wallet.cpp

```diff
@@ -3149,6 +3148,7 @@ static bool IsCurrentForAntiFeeSniping(interfaces::Chain& chain)
  */
 static uint32_t GetLocktimeForNewTransaction(interfaces::Chain& chain)
 {
+    uint32_t const height = chain.getHeight().value_or(-1);
```

Comment:
17138 how is this related?

Reply:
These 2 are aliens and just stayed in this branch after all rebases:
https://github.com/bitcoin/bitcoin/pull/17138
https://github.com/bitcoin/bitcoin/pull/17049
Not related to `rpc_createmultisig.py`

Reply:
But I don't see the changes you attribute to 17138 in that commit